conf/app.ini

```diff
@@ -439,6 +439,11 @@ SCHEDULE = @every 24h
 ; Archives created more than OLDER_THAN ago are subject to deletion
 OLDER_THAN = 24h
 
+; Synchronize external user data
```

Comment:
"Same information" being ? Email and FullName ? Anything else ?
Could it be written in the comment ?